starter_code/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Relaxr</title>
+  </head>
+  <body>
+  <img src="/Users/jeauxy/Code/WDI/homework/relaxr_html/starter_code/images/header_bg.jpg"/>
+    <h1>Get A piece of mind with a single tap</h1>
+      <button type="button">Get It Now</button>
```

Comment:
Make sure you watch your indentations. img, h1, button, h2, p, and ul are all siblings here .